docs/source/install-tutorial.rst

```diff
@@ -30,24 +43,36 @@ commands to run
   . bin/activate
   cd repo
   # install bepasty and requirements
-  pip install -e .
+  pip3 install -e .
   # add gunicorn and gevent for hosting
-  pip install gunicorn gevent
+  pip3 install gunicorn gevent
 
-config file for bepasty -- ``/home/bepasty/bepasty.conf``:
+Config file for bepasty -- ``/home/bepasty/bepasty.conf``:
 
 Copy ``src/bepasty/config.py`` to ``/home/bepasty/bepasty.conf`` first,
 remove the ``class Config`` and remove all indents in the file.
 The comments can be removed too, if you feel the need to.
-At last modify these two configs variables:
+
+An easy way to do this in one go:
+
+::
+
+  # create trim alias
+  alias trim="awk '{\$1=\$1};1'"
+  # remove the top 11 lines + trim spaces at beginning of lines
+  trim src/bepasty/config.py | tail -n +11 > /home/bepasty/bepasty.conf
```

Comment:
please do not invent such tools for an install tutorial. just describe what needs to be done.

Reply:
Maybe we can agree to put the recipe in the Tips & Tricks instead?